index.html

```diff
@@ -22,58 +22,81 @@
   <body>
     <section class="container">
       <header class="upper">
-        <nav>
+        <nav class="navigator">
+          <a href="">Hello</a>
+          <a href="">portofolio</a>
+          <a href="">About</a>
+          <a href="">Contact</a>
           <img
             class="image"
             src="assests/images/ic_menu.png"
             alt="the menu icon"
           />
         </nav>
       </header>
-      <section class="content">
-        <h1 class="my-name">Bassem Shams</h1>
-        <p class="paragraph">
-          Hello! I am a software developer! I can help you build a product,
-          feature or website. Take a look of my works. If you like what you see
-          and have a project you need coded, do not hesitate and contact me.
-        </p>
-        <button class="button">Start collaboration</button>
+      <section class="hello-content">
+        <article class="content">
+          <h1 class="my-name">Bassem Shams</h1>
+          <p class="paragraph">
+            Hello! I am a software developer! I can help you build a product,
```

Comment:
- [ ] Please place some left margins for this section to match the design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/160181531-48b1d0c9-780b-49be-a27c-7ccc842890af.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/160181576-e0fe8c17-ea8d-4265-9461-3a9b44e41f10.png)